pkgs/servers/klipper/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, python2
+}:
+stdenv.mkDerivation rec {
+  name = "klipper";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "KevinOConnor";
+    repo = "klipper";
+    rev = "v${version}";
+    sha256 = "1ijy2ij9yii5hms10914i614wkjpsy0k4rbgnm6l594gphivdfm7";
+  } + "/klippy";
```

Comment:
I think there is a `sourceDir` or `sourcePath` option that we should use instead.

Reply:
`sourceRoot`, I changed it :)